husky_extensions/itembox.js

```diff
@@ -76,7 +80,12 @@ define(function() {
                 return [
                     '<div class="white-box form-element" id="', this.ids.container, '">',
                     '    <div class="header">',
-                    '        <span class="fa-plus-circle icon left action', !!this.options.hideAddButton ? ' hidden' : '', '" id="', this.ids.addButton, '"></span>',
+                    '        <span class="', this.options.actionIcon ,' icon left action', !!this.options.hideAddButton ? ' hidden' : '', '" id="', this.ids.addButton, '"></span>',
+                    '        <span class="selected-counter', !!this.options.hideSelectedCounter ? ' hidden' : '', '">',
+                    '            <span class="num">0</span><span> ', this.sandbox.translate(this.options.translations.elementsSelected) ,'</span>',
+                    '        </span>',
+                    '        <span class="fa-cog icon right border', !!this.options.hideConfigButton ? ' hidden' : '', '" id="', this.ids.configButton, '"></span>',
+                    '        <span class="no-content">', this.sandbox.translate(this.options.translations.noContentSelected), '</span>',
```

Comment:
@marcelmoosbrugger something like "no-content-indicator" would be better. you allready use "no-content" for the container